docs/source/release-management-notes/index.md

```diff
@@ -0,0 +1,81 @@
+---
+myst:
+  html_meta:
+    "description": "Release management notes for Volto"
+    "property=og:description": "Release management notes for Volto"
+    "property=og:title": "Volto release management notes"
+    "keywords": "Volto, Plone, frontend, Release Notes, change log, changelog, change history"
+---
+
+# Release management notes
+
+This chapter describes how maintainers manage the releases of Volto.
+
```

Comment:
````suggestion

```{seealso}
Read the [Plone Release Schedule](https://plone.org/download/release-schedule) for details of how Volto's release management works in Plone.
```


````